### ISSUE TITLE
`build-config.json` when doing a custom build doesn't update links

### ISSUE THREAD

**Issue by jvanoostveen** [ckeditor/ckeditor-dev#920]
## Are you reporting a feature request or a bug?

Bug on CKEditor website - builder.
When using the builder with an uploaded `build-config.json`, the package downloaded used to have  updated links in the top comment (the (1), (2) and (3) links).
Currently, the links are not updated and while the download using the builder provides the latest version (4.7.3 for now), the links in de build-config.json - we are using the third url, straight download for optimized latest version, do **not** provide the latest, but the version we previously had (which makes sense, as the urls did not change).

### Expected result

After configuring and clicking download, I would expect the `build-config.json` to contain updated links.

### Actual result

Links are not updated and using them will download the previously configured version of CKEditor.

## Other details

* CKEditor version: 4.7.3

**Comment by jvanoostveen** [comment]
Is there any update on this? Currently, we can't update to the latest CKEditor based on a custom config without checking in the CKEditor code.
Or is there some other place where I should report this bug?

**Comment by f1ames** [comment]
@jvanoostveen, do you mean that when uploading the same `build-config` file to https://ckeditor.com/builder after new release, the generated package contains previously generated CKEditor package?

So you have downloaded `ckeditor_4.7.2_somehash.zip` and now it is `ckeditor_4.7.3_somehash.zip` (the `hash` is the same as in previous version), but the package contains `4.7.2` version?

Yes, it is not a proper place to report bugs concerning online builder, so I will rereport it in the proper one in the meantime.

**Comment by jvanoostveen** [comment]
You have it almost correct, the downloaded file had indeed 4.7.3 in it, and has a different hash than the 4.7.2. But inside the package, there is a new `build-config.json` file, which until now contained the urls for downloading this specific version and config (the urls mentioned at `(2)` and `(3)`). But since 4.7.3, the urls have not been updating, and when using the urls to download the ckeditor again, I receive version 4.7.2 instead of the expected 4.7.3.
We use these urls inside the `build-config.json` for the automated packaging; allowing us to download the correct version and config without having to add any ckeditor code to our repository.

**Comment by f1ames** [comment]
@jvanoostveen, thanks for clarification. So the build is fine, but the `build-config.js` file contains outdated links (links (2) and (3)) to a previous build with `4.7.2` release and not the current one.

One more question, is it possible for you to provide your `build-config.json` file which you are using to generate build? (you might need to change it to `.txt` as Github won't probably accept it).

**Comment by jvanoostveen** [comment]
That is correct.

The config file:
[build-config.js.txt](https://github.com/ckeditor/ckeditor-dev/files/1340262/build-config.js.txt)

**Comment by f1ames** [comment]
@jvanoostveen, I used your config to generate a custom build. In this custom build `build-config.js` file there is:

```
 * (2) http://ckeditor.com/builder/264adc8f36c6b256fef2f85264dd955e
 *     Visit online builder to build CKEditor, starting with the same setup as before.
 *
 * (3) http://ckeditor.com/builder/download/264adc8f36c6b256fef2f85264dd955e
 *     Straight download link to the latest version of CKEditor (Optimized) with the same setup as before.
```

these links are the same as in your `build-config.js` (and should be), but from what I see those two links lead to download of CKEditor 4.7.3 release package and not 4.7.2 release.

Could you recheck on your side? Or maybe I misunderstood something?

**Comment by jvanoostveen** [comment]
Hmm...
Previously, the url would change, but I might also have tweaked settings causing it to change.
I did check the download before, and it provided me with `4.7.2` (I checked the ckeditor.js source file), but when I download it now, I do get `4.7.3`. Maybe the CDN wasn't updated yet or something, as the release was quite new when I tried it.

On the other hand, if it does give me the latest version on the same url each time, our approach is flawed, as we might test our code against `4.7.2` (locally) but deliver an artifact with `4.7.3` as it is downloaded again... Have to rethink our strategy.

But for now, this issue has solved itself...

Thanks for investigating! And sorry it was nothing to solve at the end...
